_includes/vendor/anchor_headings.html

```diff
@@ -64,7 +95,7 @@
     {% capture anchor %}{% endcapture %}
 
     {% if html_id and headerLevel >= minHeader and headerLevel <= maxHeader %}
-      {% capture anchor %}href="#{{ html_id}}" aria-labelledby="{{ html_id}}"{% endcapture %}
+      {% capture anchor %}href="#{{ html_id }}"{% endcapture %}
```

Comment:
https://github.com/pmarsceill/just-the-docs/blob/c4d5f4f6b0a942e8fd067e55c35c1467d6805c43/_layouts/default.html#L104

```
{% include vendor/anchor_headings.html html=content beforeHeading="true" anchorBody="<svg viewBox=\"0 0 16 16\" aria-hidden=\"true\"><use xlink:href=\"#svg-link\"></use></svg>" anchorClass="anchor-heading" anchorAttrs="aria-labelledby=\"%html_id%\"" %}
```

In order to prevent losing the `aria-labelledby` attribute in future updates to this snippet, you can update `_layouts/default.html` to use the `anchorAttrs` parameter.

Reply:
@allejo thanks for reviewing, and for the suggested update, which I've added to the PR. I had overlooked that our copy of `anchor_headings.html` had been modified in release 0.3.1 by PR #376